python/src/DistributionImplementation_doc.i.in

```diff
@@ -589,27 +596,26 @@ OT_Distribution_computePDFGradient_doc
 Parameters
 ----------
 interval : :class:`~openturns.Interval`
-    An interval, possibly multivariate.
+    An interval in :math:`\Rset^d`.
 
 Returns
 -------
-P : float
-    Interval probability.
+p : float
+    The probability of :math:`interval`.
 
 Notes
 -----
-This computes the probability that the random vector :math:`\vect{X}` lies in
-the hyper-rectangular region formed by the vectors :math:`\vect{a}` and
-:math:`\vect{b}`:
+This computes the probability that the random vector :math:`\vect{X}` lies in :math:`interval`.
+
+If :math:`I = \bigcap\limits_{i=1}^d [a_i, b_i]`, then we have:
 
 .. math::
 
-    \Prob{\bigcap\limits_{i=1}^n a_i < X_i \leq b_i} =
-        \sum\limits_{\vect{c}} (-1)^{n(\vect{c})}
+    \Prob{\vect{X} \in I} = \sum\limits_{\vect{c}} (-1)^{n(\vect{c})}
             F_{\vect{X}}\left(\vect{c}\right)
 
-where the sum runs over the :math:`2^n` vectors such that
-:math:`\vect{c} = \Tr{(c_i, i = 1, \ldots, n)}` with :math:`c_i \in [a_i, b_i]`,
+where the sum runs over the :math:`2^d` vectors such that
+:math:`\vect{c} = \Tr{(c_i, i = 1, \ldots, d)}` with :math:`c_i \in [a_i, b_i]`,
```

Comment:
According to [math.stackexchange.com](https://math.stackexchange.com/questions/106835/generalization-to-n-dimensions-of-distribution-function-evaluation-over-an-hyp) The number $c_i$ is not the _interval_ $[a_i, b_i]$, it is in the _set_ {$a_i,b_i$}$.
```suggestion
:math:`\vect{c} = \Tr{(c_i, i = 1, \ldots, d)}` with :math:`c_i \in \{a_i, b_i\}`,
```